scripts/gcp/app/conf.toml.tmpl

```diff
@@ -20,7 +20,7 @@
 WebStaticFilesPath = "web"
 
 [AccountManager]
-Type = "GCP"
+Type = "IAP"
```

Comment:
Let's keep App Engine deployment config with `GCP` for know as there's no documentation to enable IAP for the App Engine deployment publicly available yet.